frontend/public/components/modals/alert-routing-modal.tsx

```diff
@@ -74,6 +74,7 @@ export const AlertRoutingModal: React.FC<AlertRoutingModalProps> = ({
                   type="text"
                   className="pf-c-form-control"
                   id="input-group-by"
```

Comment:
I'm actually using them [here](https://github.com/openshift/console/blob/d1e7a38b002baf1090ecf2682f7b474a50641070/frontend/public/components/modals/alert-routing-modal.tsx#L36).  I think this is an older/basic way of getting form values, I'm sure this can be rewritten with `useState` hooks if we don't want to use `id`'s

Reply:
> I'm actually using them [here](https://github.com/openshift/console/blob/d1e7a38b002baf1090ecf2682f7b474a50641070/frontend/public/components/modals/alert-routing-modal.tsx#L36). I think this is an older/basic way of getting form values, I'm sure this can be rewritten with `useState` hooks if we don't want to use `id`'s

Yeah, I wouldn't recommend writing the form that way. `useState` seems a lot cleaner.